java/configuration/src/main/java/sleeper/configuration/properties/table/TablePropertyImpl.java

```diff
@@ -86,6 +87,11 @@ public String getDescription() {
         return description;
     }
 
+    @Override
+    public PropertyGroup getPropertyGroup() {
+        return PropertyGroup.COMMON;
+    }
```

Comment:
The table properties have their own separate set of groups. If that's a separate issue, we should change this PR to only modify the instance properties and the instance property report, and raise that as its own issue.

Reply:
I think that makes sense, maybe have InstancePropertyGroup and TablePropertyGroup. I'll revert the grouping changes in the TableProperty classes

Reply:
It seems fine to split out InstancePropertyGroup and TablePropertyGroup in the other issue.